packages/dendron-viz/src/language-colors.json

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
Do we need this file? It's pretty big and I don't think it makes sense for us since we're not visualizing different programming languages. We can look at how to do colors later.

If it is hard to remove, it's fine to keep it for now.